### PR TITLE
Update professional-development.md

### PR DESCRIPTION
**Pull request by ContentBrewmaster**
In consultation with Rachael, updated instructions for prodev sprints (removing ref to Jason Heaster), added link to Travel info for expenses and moved the historical prodev amounts to an accordion. Also applied sentence case to headings for consistency.

----
:books: Documentation preview :books:: https://civicactions-handbook--1036.org.readthedocs.build/en/1036/

### CODE REVIEW

**Comment by ContentBrewmaster** [comment]
[Update Prodev page](https://trello.com/c/J2KSrDR1/270-update-prodev-page)

File: employee-benefits/professional-development.md [hunk 1]
```diff
@@ -27,45 +26,45 @@ Historical Prodev Hours Spent:
 | 2018 | 1736                           | 44                            | 39.5                                          |
 | 2019 | 1668                           | 56                            | 29.8                                          |
 
-_\*Snapshot from February 12, 2020._
+_\*Snapshot from February 12, 2020._</details>
```
Comment (dmundra) [inline employee-benefits/professional-development.md]: @ContentBrewmaster looks like the details displays works but the table inside of it is not rendered. Might just need some tweaking to make it work. See preview https://civicactions-handbook--1036.org.readthedocs.build/en/1036/employee-benefits/professional-development/

Comment (ContentBrewmaster) [reply employee-benefits/professional-development.md]: @dmundra Could it be that we need to change our mkdocs.yml file?
https://squidfunk.github.io/mkdocs-material/reference/data-tables/#configuration

Comment (dmundra) [reply employee-benefits/professional-development.md]: Good question. The table of the practice area does work https://civicactions-handbook--1036.org.readthedocs.build/en/1036/practice-areas/about-practice-areas/. Might need some more R&D.

Comment (dmundra) [reply employee-benefits/professional-development.md]: Looks like we need another plugins from https://github.com/squidfunk/mkdocs-material/issues/4084 called 'Markdown in HTML' see https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#markdown-in-html. I will push up that change.

**Comment by dmundra** [comment]
@grugnog since I pushed up a change here, can you review it?

**Comment by grugnog** [comment]
@ContentBrewmaster @dmundra this looks good - I wonder if there is a possibility to improve some of the long/complex sentences identified by the checks though?

**Comment by ContentBrewmaster** [comment]
Agree @grugnog that a full rewrite is needed. This was an interim edit to remove inaccurate info (reference to Jason) and support more findability for travel-related expenses. We could pause this until I return and can work with Marc/Rachel on a more comprehensive edit, or release this version now.

**Comment by dmundra** [comment]
@grugnog looks like remark lint ran but didn't report the suggestions like before. See warning here https://github.com/CivicActions/guidebook/actions/runs/4086895319/jobs/7046744378

> Warning: Unexpected input(s) 'github_token', 'command', valid inputs are ['entryPoint', 'args']

**Comment by dmundra** [comment]
~Fixed the remark lint warning but not seeing the inline suggestions appear in the diff view (files tab).~

Issue fixed by @grugnog and my change reverted! Thank you!

**Comment by grugnog** [comment]
@dmundra looks like this is fixed - we have complaints about hard to understand sentences again :)

**Review by marctjones — APPROVED** [review]
Looks good. As Denise said this is definitely a page that needs to be revisited overall. But getting the factually wrong information out of there is a good quick win.

**Review by dmundra — APPROVED** [review]

**Comment by dmundra** [comment]
Thanks @marctjones!

**Pull request merged**